reposerver/repository/repository.go

```diff
@@ -806,6 +806,14 @@ func findManifests(appPath string, repoRoot string, env *v1alpha1.Env, directory
 		if !manifestFile.MatchString(f.Name()) {
 			return nil
 		}
+
+		for _, exclusion := range directory.Exclusions {
```

Comment:
It is better to use "glob" expression instead of Regex since we are using it to filter file paths. Please use `Match` from `util/glob` package.